brayns/parameters/GeometryParameters.cpp

```diff
@@ -152,6 +153,8 @@ GeometryParameters::GeometryParameters()
         "Minimum and maximum values for the simulation [float float]")(
         PARAM_CIRCUIT_SIMULATION_HISTOGRAM_SIZE.c_str(), po::value<size_t>(),
         "Number of values defining the simulation histogram [int]")(
+        PARAM_CIRCUIT_RANDOM_SEED.c_str(), po::value<size_t>(),
```

Comment:
I'd use const char* for all these strings if you really need the C representation in the end

Reply:
Seems reasonable, but that is for another PR.